docs/multiplayer-games.md

```diff
@@ -12,6 +12,7 @@ Try these multiplayer games with your friends!
 * imageUrl: /static/multiplayer/perfect-fit.png
 * url: https://arcade.makecode.com/---multiplayer?host=_bz3CCuWFiepH
 * cardType: link
+* otherAction: https://arcade.makecode.com/_bz3CCuWFiepH, , sharedExample
```

Comment:
I purposely don't include the editor info in `otherAction` (hence the extra comma) because having the editor info makes the icon different, which doesn't look good with the icon for hosting the game:
![image](https://user-images.githubusercontent.com/15070078/214309822-2af71269-53ce-4fa3-9d6d-0ed2d8edb9e4.png)